From 13f72e894c7c148c0e5f754b54f0284cb13e5d56 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 11:39:15 -0400
Subject: [ethernet] i40e/i40evf: Bump i40e to 0.4.13 and i40evf to 0.9.35

Message-id: <1409658042-22689-195-git-send-email-sassmann@redhat.com>
Patchwork-id: 90392
O-Subject: [RHEL7.1 PATCH 194/281] i40e/i40evf: Bump i40e to 0.4.13 and i40evf to 0.9.35
Bugzilla: 1091126
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Catherine Sullivan <catherine.sullivan@intel.com>

Bump versions.

Change-ID: Ifaed5404b9e953a11f4c88953ffe4bc8937705f1
Signed-off-by: Catherine Sullivan <catherine.sullivan@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 7974d5e5ed88f4214b0d931e96040e45d2db1e88)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_main.c b/drivers/net/ethernet/intel/i40e/i40e_main.c
index bf2fcde..0e2f3da 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_main.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_main.c
@@ -39,7 +39,7 @@ static const char i40e_driver_string[] =
 
 #define DRV_VERSION_MAJOR 0
 #define DRV_VERSION_MINOR 4
-#define DRV_VERSION_BUILD 10
+#define DRV_VERSION_BUILD 13
 #define DRV_VERSION __stringify(DRV_VERSION_MAJOR) "." \
       __stringify(DRV_VERSION_MINOR) "." \
       __stringify(DRV_VERSION_BUILD)    DRV_KERN
-- 
1.7.1